test/integration/app-dir-export/test/index.test.ts

```diff
@@ -0,0 +1,176 @@
+    await waitFor(delay)
+
+    expect(await browser.elementByCss('h1').text()).toBe('Another')
+    expect(await browser.elementByCss(a(1)).text()).toBe('Visit the home page')
+    await browser.elementByCss(a(1)).click()
+    await waitFor(delay)
+
+    expect(await browser.elementByCss('h1').text()).toBe('Home')
+    expect(await browser.elementByCss(a(2)).text()).toBe(
+      'another has trailingslash'
+    )
+    await browser.elementByCss(a(2)).click()
+    await waitFor(delay)
+
+    expect(await browser.elementByCss('h1').text()).toBe('Another')
+    expect(await browser.elementByCss(a(1)).text()).toBe('Visit the home page')
+    await browser.elementByCss(a(1)).click()
+    await waitFor(delay)
+
+    expect(await browser.elementByCss('h1').text()).toBe('Home')
+    expect(await browser.elementByCss(a(3)).text()).toBe('another first page')
+    await browser.elementByCss(a(3)).click()
+    await waitFor(delay)
+
+    expect(await browser.elementByCss('h1').text()).toBe('first')
+    expect(await browser.elementByCss(a(1)).text()).toBe('Visit another page')
+    await browser.elementByCss(a(1)).click()
+    await waitFor(delay)
+
+    expect(await browser.elementByCss('h1').text()).toBe('Another')
+    expect(await browser.elementByCss(a(4)).text()).toBe('another second page')
+    await browser.elementByCss(a(4)).click()
+    await waitFor(delay)
+
+    expect(await browser.elementByCss('h1').text()).toBe('second')
+    expect(await browser.elementByCss(a(1)).text()).toBe('Visit another page')
+    await browser.elementByCss(a(1)).click()
+    await waitFor(delay)
+
+    expect(await browser.elementByCss('h1').text()).toBe('Another')
+    expect(await browser.elementByCss(a(5)).text()).toBe('image import page')
+    await browser.elementByCss(a(5)).click()
+    await waitFor(delay)
+
+    expect(await browser.elementByCss('h1').text()).toBe('Image Import')
+    expect(await browser.elementByCss(a(2)).text()).toBe('View the image')
+    expect(await browser.elementByCss(a(2)).getAttribute('href')).toContain(
+      '/test.3f1a293b.png'
+    )
+  } finally {
+    await stopApp(app)
+    nextConfig.restore()
+    slugPage.restore()
+  }
+}
+
+describe('app dir with next export', () => {
+  it.each([{ trailingSlash: false }, { trailingSlash: true }])(
+    "should work with trailingSlash '$trailingSlash'",
+    async ({ trailingSlash }) => {
+      await runTests({ trailingSlash })
+    }
+  )
+  it.each([
+    { dynamic: 'undefined' },
+    { dynamic: "'error'" },
+    { dynamic: "'force-static'" },
+  ])('should work with dynamic $dynamic', async ({ dynamic }) => {
+    await runTests({ dynamic })
+    const opts = { cwd: exportDir, nodir: true }
+    const files = ((await glob('**/*', opts)) as string[])
+      .filter((f) => !f.startsWith('_next/static/chunks/'))
+      .sort()
+    expect(files).toEqual([
+      '404.html',
+      '404/index.html',
+      '_next/static/media/favicon.603d046c.ico', // TODO: should this be here?
```

Comment:
I'll handle this issue later
```suggestion
      '_next/static/media/favicon.603d046c.ico', // TODO-METADATA: this should not be generated
```

Reply:
Fixed in [1aa1d58](https://github.com/vercel/next.js/pull/47022/commits/1aa1d580ee3e625f1d17c02ecc3a914b9b10df2e)